prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -96,8 +96,6 @@ postsubmits:
         - "/main.sh"
         args: ["gcp-guest", "us-west1-a", "deb9,el6,el7,el8,goo", "gs://gcp-guest-package-uploads/osconfig/"]
     max_concurrency: 10
-    branches:
```

Comment:
can you explain why you've removed this?

Reply:
It's a duplicate, see a few lines above